.vscode/settings.json

```diff
@@ -0,0 +1,10 @@
+{
```

Comment:
Are you sure about adding your `.vscode` folder as part of this?

Reply:
Yeah, it's exactly what I was gonna say.

Reply:
> Are you sure about adding your `.vscode` folder as part of this?

HAHAAHHA DAMN FORGOT :laughing:

Reply:
whoops, fixed now :)